pkg/server/server.go

```diff
@@ -359,9 +359,13 @@ func (s *Server) Check(ctx context.Context, req *openfgav1.CheckRequest) (*openf
 		TupleKey:             req.GetTupleKey(),
 		ContextualTuples:     req.ContextualTuples.GetTupleKeys(),
 		ResolutionMetadata: &graph.ResolutionMetadata{
-			Depth: s.resolveNodeLimit,
+			Depth:              s.resolveNodeLimit,
+			DatastoreCallCount: 0,
 		},
 	})
+
+	grpc_ctxtags.Extract(ctx).Set("datastore_call_count", int64(resp.ResolutionMetadata.DatastoreCallCount))
+	span.SetAttributes(attribute.Int64("datastore_call_count", int64(resp.ResolutionMetadata.DatastoreCallCount)))
```

Comment:
Can we please rename to `datastore_query_count` to be consistent with other vernacular we've established in metric names etc..?